connext_cmake_module/cmake/Modules/FindConnext.cmake

```diff
@@ -311,7 +311,10 @@ endif()
 
 if(Connext_FOUND)
   if(NOT WIN32)
-    list(APPEND Connext_LIBRARIES "pthread" "dl")
+      list(APPEND Connext_LIBRARIES "dl")
+      if(NOT ANDROID)
+        list(APPEND Connext_LIBRARIES "pthread" "dl")
+      endif()
```

Comment:
Please unindent all four lines by one level / two spaces to match the style of the file.

Reply:
done